client/assets/sass/styles.sass

```diff
@@ -4,7 +4,6 @@
 @import '_bem-support/index'
 
 // Variables: colors, sizes, ...
-@import 'colors'
 @import 'app_colors'
 @import 'app_urls'
 
```

Comment:
Hmm, for modules that are repeatedly required is there any greater convenience we can provide?